README.md

```diff
@@ -96,6 +96,15 @@ Support for packages managers is still work in progress and expected
 to be finished for one of the upcoming release. For details about the
 progress follow the [issue #471][17]
 
+#### Archlinux
```

Comment:
Very tiny nit: s/Archlinux/Arch Linux (I think this is their official naming or at least what they use on the website)

Reply:
```suggestion
#### Arch Linux
```